keyboards/iris/keymaps/vyolle_steno/rules.mk

```
@@ -1,7 +1,7 @@
-EXTRAKEY_ENABLE = no
-MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
-STENO_ENABLE = yes          # Additional protocols for Stenography(+1700), requires VIRTSER
+EXTRAKEY_ENABLE = yes       # Disable if you run out of usb endpoints. Media keys need this to work.
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700) Uses extra usb enpoints. 
```

Tiny, tiny typo here in the comment.

```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys(+4700) Uses extra usb endpoints. 
```